### PR TITLE
config: use millis for all interval

### PR DESCRIPTION
**Pull request by BusyJay**
tikv-server will convert all readable int to millis, so we should treat all intervals as millis.

@siddontang @zhangjinpeng1987 @disksing  PTAL

### CODE REVIEW

File: etc/config-template.toml [hunk 1]
```diff
@@ -75,14 +75,14 @@ split-region-check-tick-interval = "5s"
 max-peer-down-duration = "5m"
 
 # Interval to check whether start compaction for a region.
-# region-compact-check-interval-secs = 300
+# region-compact-check-interval = "5min"
 # When delete keys of a region exceeds the size, a compaction will be started.
 # region-compact-delete-keys-count = 200000
 # Interval to check whether start lock compaction for a region.
-# lock-cf-compact-interval-secs = 600
+# lock-cf-compact-interval = "60s"
```
Comment (zhangjinpeng1987) [inline etc/config-template.toml]: 10min

File: src/raftstore/store/config.rs [hunk 2]
```diff
@@ -30,24 +30,24 @@ const SPLIT_REGION_CHECK_TICK_INTERVAL: u64 = 10000;
...
+const PD_STORE_HEARTBEAT_TICK_INTERVAL: u64 = 10000;
 const STORE_CAPACITY: u64 = u64::MAX;
 const DEFAULT_NOTIFY_CAPACITY: usize = 4096;
-const DEFAULT_MGR_GC_TICK_INTERVAL_MS: u64 = 60000;
+const DEFAULT_MGR_GC_TICK_INTERVAL: u64 = 60000;
 const DEFAULT_SNAP_GC_TIMEOUT_SECS: u64 = 60 * 10;
 const DEFAULT_MESSAGES_PER_TICK: usize = 256;
 const DEFAULT_MAX_PEER_DOWN_SECS: u64 = 300;
-const DEFAULT_LOCK_CF_COMPACT_INTERVAL_SECS: u64 = 60 * 10; // 10 min
+const DEFAULT_LOCK_CF_COMPACT_INTERVAL: u64 = 60 * 10 * 1000; // 10 min
```
Comment (disksing) [inline src/raftstore/store/config.rs]: `10 * 60 * 1000` is more readable.

**Comment by ngaut** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

File: etc/config-template.toml [hunk 3]
```diff
@@ -75,14 +75,14 @@ split-region-check-tick-interval = "5s"
...
+# region-compact-check-interval = "5m"
 # When delete keys of a region exceeds the size, a compaction will be started.
 # region-compact-delete-keys-count = 200000
 # Interval to check whether start lock compaction for a region.
-# lock-cf-compact-interval-secs = 600
+# lock-cf-compact-interval = "10m"
 
 # Interval (s) to check region whether the data are consistent.
-# consistency-check-interval-secs = 0
+# consistency-check-interval = 0
```
Comment (zhangjinpeng1987) [inline etc/config-template.toml]: what 0 means? off?

Comment (BusyJay) [inline etc/config-template.toml]: Yes.

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by siddontang** [comment]
CI failed

**Comment by BusyJay** [comment]
circleci fails to fetch the code, @andelf PTAL

**Comment by andelf** [comment]
@BusyJay  it says ``Project pingcap/tikv isn't configured with a deploy key or a github user, so we may not be able to test all pushes.``
Project owner should click ``Add SSH key`` on https://circleci.com/gh/pingcap/tikv/11

**Comment by BusyJay** [comment]
@andelf @shenli @siddontang Please note that after this PR, some names of configuration entries have been changed.

**Pull request merged**